### ISSUE TITLE
Unrecognized tag: lasso-page

### ISSUE THREAD

**Issue by charlesread** [charlesread/hmls#1]
When using `lasso` to bundle resources the following error is thrown.

```
Debug: internal, implementation, error 
    Error: Uncaught error: An error occurred while trying to compile template at path "/Users/cread/Development/node/hmls-test/pages/slash/index.marko". Error(s) in template:
1) [pages/slash/index.marko:1:0] Unrecognized tag: lasso-page - More details: https://github.com/marko-js/marko/wiki/Error:-Unrecognized-Tag
2) [pages/slash/index.marko:10:4] Unrecognized tag: lasso-head - More details: https://github.com/marko-js/marko/wiki/Error:-Unrecognized-Tag
3) [pages/slash/index.marko:16:0] Unrecognized tag: lasso-body - More details: https://github.com/marko-js/marko/wiki/Error:-Unrecognized-Tag
```

**Comment by charlesread** [comment]
While a permanent resolution is not yet available a simple workaround is to simply install `@lasso/marko-taglib` into your project (see [https://github.com/lasso-js/lasso-marko/issues/14](https://github.com/lasso-js/lasso-marko/issues/14))

```
npm install --save @lasso/marko-taglib
```

**Comment by charlesread** [comment]
To my knowledge this has been fixed as of the 2.1.x releases, if not please reopen this issue.
